core/templates/pages/story-editor-page/navbar/story-editor-navbar.component.html

```diff
@@ -1,30 +1,36 @@
 <ul class="nav navbar-nav oppia-navbar-nav navbar-desktop float-right">
   <li class="oppia-dropdown-menu-container">
     <div ngbDropdown>
-      <button class="btn btn-light dropdown oppia-save-changes-button float-left e2e-test-save-story-button"
-              [ngClass]="{'btn-success': isStorySaveable()}"
-              (click)="saveChanges()"
-              [disabled]="!isStorySaveable()">
-        <span *ngIf="!isSaveInProgress()">
-          <i class="fas fa-save md-18 md-dark oppia-save-publish-button-icon story-editor-save-draft-icon"
-             alt="Save Story">
-          </i>
-          <span class="oppia-save-publish-button-label oppia-save-draft-text float-left">
-            Save Draft
-          </span>
-          <span class="d-none d-md-block float-left oppia-changes-count"
-                *ngIf="getChangeListLength() > 0">
-            ({{ getChangeListLength() }})
+      <span [matTooltip]="validationIssues.concat(explorationValidationIssues).concat(prepublishValidationIssues).join('\n')"
+            matTooltipClass="oppia-mat-tooltip-list" [matTooltipDisabled]="isWarningTooltipDisabled()"
+            aria-label="List that shows issues preventing to save story as draft" class="w-100">
+        <button class="btn btn-light dropdown oppia-save-changes-button float-left e2e-test-save-story-button"
+                [ngClass]="{'btn-success': isStorySaveable()}"
+                (click)="saveChanges()"
+                [disabled]="!isStorySaveable()">
+          <span *ngIf="!isSaveInProgress()">
+            <i *ngIf="isWarningTooltipDisabled()"
+               class="fas fa-save md-18 md-dark oppia-save-publish-button-icon story-editor-save-draft-icon"
+               alt="Save Story">
+            </i>
+            <mat-icon *ngIf="!isWarningTooltipDisabled()" class="story-save-draft-warning">
+              warning
+            </mat-icon>
```

Comment:
Do not use Material icon use Font Awesome icon instead.

Reply:
I followed the existing code from blog-post-editor.component.html and used mat-icon. Is there a specific reason for not using mat-icon?

Reply:
We are trying to deprecate Material icons from our codebase. So we should not add new ones.

Reply:
Changed to Font Awesome icon